### DISCUSSION TITLE
Using a proxy to access tmdb request 400

### DISCUSSION THREAD

**Discussion by sakura-fly** [seerr-team/seerr#746] [Q&A]
I can't access tmdb, so use proxy.
build by source,configure the proxy through the Environment file. after run, failed to sync from jellyfin . read log, "[TMDB] Failed to find by external ID: Request failed with status code 400" was found.
The log:
`2024-05-08T08:20:00.896Z [error][Jellyfin Sync]: Failed to process Jellyfin item. Id: 89a249a33b8724a82cb28594d51a947a {"errorMessage":"[TMDB] Failed to get TV show using the external TVDB ID: [TMDB] Failed to find by external ID: Request failed with status code 400","jellyfinitem":{"Name":"夜晚的水母不会游泳","ServerId":"9c14569ea1424102bd28a9b5719c516b","Id":"89a249a33b8724a82cb28594d51a947a","PremiereDate":"2024-04-07T00:00:00.0000000Z","OfficialRating":"TV-14","ChannelId":null,"CommunityRating":8.7,"RunTimeTicks":14400000000,"ProductionYear":2024,"IsFolder":true,"Type":"Series","UserData":{"UnplayedItemCount":5,"PlaybackPositionTicks":0,"PlayCount":0,"IsFavorite":false,"Played":false,"Key":"432808"},"ChildCount":3,"Status":"Continuing","AirDays":[],"ImageTags":{"Primary":"d9105bfeaf5bae9156cb9c712543cd32","Logo":"405367e7b6ce7470738a9f26fc0fa728"},"BackdropImageTags":["10a150f1e1bb8682a8a482bd76fa886b"],"ImageBlurHashes":{"Backdrop":{"10a150f1e1bb8682a8a482bd76fa886b":"WbFY4OV[IoIooz%M~XaLt7M|WVt7%MRjxuM{NGxa%gIoRjaKj=t6"},"Primary":{"d9105bfeaf5bae9156cb9c712543cd32":"dJH-#=--J5-r%y%LD+9aE1IV9H9Z~qE1t5xa%zIpM{W9"},"Logo":{"405367e7b6ce7470738a9f26fc0fa728":"H8O49%ozs%_4~qMx%2j[_3?bM{M{oeM{xvRj9Ft8"}},"LocationType":"FileSystem"}}`

the Environment file
`
PORT=5055

HTTP_PROXY="http://127.0.0.1:8889/"
HTTPS_PROXY="http://127.0.0.1:8889/"
no_proxy="localhost,127.0.0.1"
`

**Comment by fallenbagel** [comment]
Are you able to hop onto our discord to troubleshoot this?
